src/actions_ls.rs

```diff
@@ -212,7 +228,7 @@ impl ActionHandler {
         let vfs: &Vfs = &self.vfs;
         let result: Vec<CompletionItem> = panic::catch_unwind(move || {
             let pos = adjust_vscode_pos_for_racer(params.position);
-            let file_path = &params.textDocument.file_name();
+            let file_path = &uri_string_to_file_name(&params.text_document.uri);
```

Comment:
I just meant that the `uri` field should not be restricting to `file://`, it is perfectly fine for `rls` to just handle `file://` but in general it could be an identifier for something else.